Content.Client/Administration/AdminNameOverlay.cs

```diff
@@ -41,7 +39,10 @@ protected override void Draw(in OverlayDrawArgs args)
                 }
 
                 // if not on the same map, continue
-                if (IoCManager.Resolve<IEntityManager>().GetComponent<TransformComponent>(entity).MapID != _eyeManager.CurrentMap)
+                if (IoCManager
+                        .Resolve<IEntityManager>()
+                        .GetComponent<TransformComponent>(entity)
+                        .MapID != args.Viewport.Eye!.Position.MapId)
```

Comment:
IEntityManager is already a dependency (I know it's on master but that was auto refactored)

Reply:
I'll just push it to master coz I CBF waiting.